Samples/src/PythonInterop/tomography-sample.ipynb

```diff
@@ -29,482 +29,238 @@
-       "\n",
-       "                    <tr><td></td><td><pre>Operation`2 get_BaseOp()</pre></td></tr>\n",
-       "<tr><td></td><td><pre>Func`2 get_Mapper()</pre></td></tr>\n",
-       "<tr><td></td><td><pre>IApplyData __dataIn(Qubit<> data)</pre></td></tr>\n",
-       "<tr><td></td><td><pre>IApplyData __dataOut(QVoid<> data)</pre></td></tr>\n",
-       "<tr><td></td><td><pre>Func`2 get_Body()</pre></td></tr>\n",
-       "<tr><td></td><td><pre>Func`2 get_AdjointBody()</pre></td></tr>\n",
-       "<tr><td></td><td><pre>Func`2 get_ControlledBody()</pre></td></tr>\n",
-       "<tr><td></td><td><pre>Func`2 get_ControlledAdjointBody()</pre></td></tr>\n",
-       "<tr><td></td><td><pre>String ToString()</pre></td></tr>\n",
-       "<tr><td></td><td><pre>String __qsharpType()</pre></td></tr>\n",
-       "<tr><td></td><td><pre>AdjointedOperation`2 get_Adjoint()</pre></td></tr>\n",
-       "<tr><td></td><td><pre>ControlledOperation`2 get_Controlled()</pre></td></tr>\n",
-       "<tr><td></td><td><pre>QVoid Apply(Qubit<> a)</pre></td></tr>\n",
-       "<tr><td></td><td><pre>T Partial(Object<> partialInfo)</pre></td></tr>\n",
-       "<tr><td></td><td><pre>PartialApplication`3 Partial(Func`2<P, Microsoft.Quantum.Simulation.Core.Qubit> mapper)</pre></td></tr>\n",
-       "<tr><td></td><td><pre>GenO Apply(Object<> args)</pre></td></tr>\n",
-       "<tr><td></td><td><pre>IOperationFactory get_Factory()</pre></td></tr>\n",
-       "<tr><td></td><td><pre>Boolean Equals(Object<> obj)</pre></td></tr>\n",
-       "<tr><td></td><td><pre>Int32 GetHashCode()</pre></td></tr>\n",
-       "<tr><td></td><td><pre>Type GetType()</pre></td></tr>\n",
-       "                </tr>\n",
-       "            \n",
-       "        </table>\n",
-       "        "
-      ],
-      "text/plain": [
-       "<WrappedCLRObject(Microsoft.Quantum.Simulation.Core.PartialApplication`3[[Microsoft.Quantum.Simulation.Core.Qubit, Microsoft.Quantum.Simulation.Core, Version=0.3.1809.2802, Culture=neutral, PublicKeyToken=40866b40fd95c7f5],[System.ValueTuple`2[[System.Double, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089],[Microsoft.Quantum.Simulation.Core.Qubit, Microsoft.Quantum.Simulation.Core, Version=0.3.1809.2802, Culture=neutral, PublicKeyToken=40866b40fd95c7f5]], mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089],[Microsoft.Quantum.Simulation.Core.QVoid, Microsoft.Quantum.Simulation.Core, Version=0.3.1809.2802, Culture=neutral, PublicKeyToken=40866b40fd95c7f5]]) at 0x1a842f7da20>"
-      ]
-     },
-     "execution_count": 11,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
-   "source": [
-    "noisy_h = qsim.run(NoisyHadamardChannel, 0.1).result()\n",
-    "noisy_h"
-   ]
-  },
-  {
-   "cell_type": "markdown",
+   "execution_count": 10,
    "metadata": {},
+   "outputs": [],
    "source": [
-    "## Tomography ##"
+    "experiment = qsharp.compile(\"\"\"\n",
```

Comment:
Should we move the definition of the operation into the `Operations.qs`? I don't see the need to compile it on the fly.

Reply:
My thought was mainly to demonstrate on-the-fly compiling, but I agree it's not technically necessary in this case.